packages/NodeNameResolver/NodeNameResolver.php

```diff
@@ -198,27 +193,18 @@ public function isStringName(string $resolvedName, string $desiredName): bool
             return $desiredName === $resolvedName;
         }
 
-        $containsWildcard = false;
-        foreach(self::REGEX_WILDCARD_CHARS as $char) {
-            if (str_contains($desiredName, $char)) {
-                $containsWildcard = true;
-                break;
-            }
-        }
-
-        if ($containsWildcard) {
-            // is probably regex pattern
-            if ($this->regexPatternDetector->isRegexPattern($desiredName)) {
-                return StringUtils::isMatch($resolvedName, $desiredName);
-            }
+        return strtolower($resolvedName) === strtolower($desiredName);
+    }
 
-            // is probably fnmatch
-            if (\str_contains($desiredName, '*')) {
-                return fnmatch($desiredName, $resolvedName, FNM_NOESCAPE);
-            }
+    public function matchesStringName(string $resolvedName, string $desiredNamePattern): bool
```

Comment:
The `$resolvedName` should accept both string and identifier IMO.

Reply:
done